common/identity.go

```diff
@@ -97,6 +97,13 @@ func SetFlorenceHeader(ctx context.Context, r *http.Request) {
 	}
 }
 
+// AddFlorenceHeader sets the given user access token (florence token) token on the given request
+func AddFlorenceHeader(r *http.Request, userAccessToken string) {
```

Comment:
I think we need to decide on a name for this and enforce consistency from here on out. For example here the method is named for `FlorenceHeader` but the parameter name is `userAccessToken`. 

I am not precious about what we call it but we need to be consistent.